### DISCUSSION TITLE
Possibility for adding an option for temporarily disabling color change for 15 minutes

### DISCUSSION THREAD

**Discussion by IceFire23** [Tyrrrz/LightBulb#345] [General]
Hello,

I was wondering if maybe there could be a new option added to temporarily disable the color change for 15 minutes? Or alternatively, maybe adding a way for users to customize what the options are for temporarily disabling the color change.

I did a few searches to see if there was already a discussion or issue discussing this but didn't find one with the keywords I used.

**Comment by Tyrrrz** [comment]
If you right click the tray icon, there are options to disable LightBulb for set periods of time

**Comment by IceFire23** [comment]
I've been using that right-click option in the tray menu, it's been much appreciated! I think I didn't include enough info in the original post. I have been using that but there is a jump in the time options there from 5 minutes to 30 minutes. 

I was wondering if it was possible for there to be a 15 minute option added that would be midway between 5 and 30 minutes. (or alternatively if perhaps there could a way for the user to customize which time options are available in the menu)

**Reply by Tyrrrz** [reply]
I'll add a 15 minute option

**Reply by IceFire23** [reply]
Thanks, appreciate it!!

**Reply by IceFire23** [reply]
I see the PR got merged in https://github.com/Tyrrrz/LightBulb/pull/346 - thanks for that! I was curious what your normal frequency is for releasing new app versions, to know when I might expect a version to be released with the new 15 minute option.

**Reply by Tyrrrz** [reply]
Sorry, I forgot about that. I'll try to get around to it next week as I wanted to include a few other things in it.

**Reply by IceFire23** [reply]
Sounds great, thanks!
